src/status_im/common/scalable_avatar/view.cljs

```diff
@@ -21,14 +20,15 @@
                                                             header-extrapolation-option)
         image-side-margin-animation (reanimated/interpolate scroll-y
                                                             scroll-animation-input-range
-                                                            [0 -20]
+                                                            [-4 -20]
                                                             header-extrapolation-option)
         theme                       (quo.theme/get-theme)]
```

Comment:
this use of theme is correct? doesn't need to get it from context theme?

Reply:
I moved it it own file but in settings header i need to use user theme for border, not the screen theme